### PR TITLE
generic: replace spi Silicon Labs patch with upstream version

### PR DESCRIPTION
**Pull request by namiltd**
Move the patches to the correct backport folder

### CODE REVIEW

**Comment by PolynomialDivision** [comment]
Thanks. Would be nice if you than could also replace the patches by their accepted upstream equivalent.

**Comment by namiltd** [comment]
Done.

**Comment by namiltd** [comment]
@dangowrt Please review and accept.

**Comment by dangowrt** [comment]
LGTM. Generally it's preferred to generate the patches from upstream Linux git checkout using `git format-patch` without any receivers, but `From` header indicating commit hash upstream. It's just how we usually do it for accepted patches, but that habit not being documented anywhere makes me lack good arguments to ask you to do that :wink:

**Comment by namiltd** [comment]
@dangowrt If you can, do it properly instead of this Pull Request, I have no experience with this.

**Comment by Ansuel** [comment]
@namiltd honestly i would just squash the 2 thing

**Comment by namiltd** [comment]
@Ansuel Done

File: target/linux/generic/backport-6.1/412-v6.3-02-spidev-Add-Silicon-Labs-SI3210-device-compatible.patch [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+Subject: [PATCH 2/2] spidev: Add Silicon Labs SI3210 device compatible
+Date: Tue, 27 Dec 2022 09:10:08 -0500	[thread overview]
+Message-ID: <20221227141011.111410-2-vincent@vtremblay.dev> (raw)
+In-Reply-To: <20221227141011.111410-1-vincent@vtremblay.dev>
+========
+From 6c9d1fd52956c3148e847a214bae9102b1811de5 Mon Sep 17 00:00:00 2001
+From: Vincent Tremblay <vincent@vtremblay.dev>
+Date: Tue, 27 Dec 2022 09:10:08 -0500
+Subject: [PATCH] spidev: Add Silicon Labs SI3210 device compatible
+>>>>>>>> a2b96889a4fbbc66c1dcf088015e6fe5e401083b:target/linux/generic/backport-6.1/412-v6.3-02-spidev-Add-Silicon-Labs-SI3210-device-compatible.patch
```
Comment (Ansuel) [inline target/linux/generic/backport-6.1/412-v6.3-02-spidev-Add-Silicon-Labs-SI3210-device-compatible.patch:25]: You messed up the rebase i think.

**Comment by namiltd** [comment]
@Ansuel Corrected

**Pull request merged**